CMakeLists.txt

```diff
@@ -3,7 +3,9 @@
 #
 
 cmake_minimum_required(VERSION 2.6)
-
+if(CMAKE_VERSION VERSION_GREATER 2.8.12)
+  cmake_policy(SET CMP0048 OLD)
+endif()
```

Comment:
Done. The warning actually only occurs if a superproject set the policy to NEW. I changed it now to  accordingly to supply the VERSION information.